src/test/java/de/tum/in/ase/parser/IntegrationTest.java

```diff
@@ -16,69 +22,87 @@
+    /**
+     * Compares the parsed JSON report with the expected JSON report
+     * @param toolGeneratedReportFileName The name of the file contains the report as generated by the different tools
+     * @param expectedJSONReportFileName  The name of the file that contains the parsed report
+     * @throws ParserException If an exception occurs that is not already handled by the parser itself, e.g. caused by the json-parsing
+     */
+    private void testParserWithFile(String toolGeneratedReportFileName, String expectedJSONReportFileName) throws ParserException, IOException {
+        File toolReport = new File(String.valueOf(REPORTS_FOLDER_PATH.resolve(toolGeneratedReportFileName)));
```

Comment:
```suggestion
        File toolReport = REPORTS_FOLDER_PATH.resolve(toolGeneratedReportFileName).toFile();
```
I think the `toFile()` method should be available in Java 8 and would make the conversion Path → File a lot easier.

Reply:
Oh, that is very convenient. Thanks for the suggestion.